planner/core/rule_predicate_push_down.go

```diff
@@ -70,8 +137,19 @@ func splitSetGetVarFunc(filters []expression.Expression) ([]expression.Expressio
 	return canBePushDown, canNotBePushDown
 }
 
+// eliminateGetVar eliminates GetVar functions if they don't conflict.
+func eliminateGetVar(exprs []expression.Expression) []expression.Expression {
```

Comment:
`eliminateGetVar` is not clear to me. I prefer names like `convertGetVar2Constant`.